STARTERKIT/gulpfile.js

```diff
@@ -188,7 +201,7 @@ gulp.task('styleguide:kss-example-chroma', function () {
     .pipe($.replace(/(\/\*|\*\/)\n/g, ''))
     .pipe($.rename('kss-example-chroma.twig'))
     .pipe($.size({showFiles: true}))
-    .pipe(gulp.dest(options.theme.build + 'twig'));
+    .pipe(gulp.dest(options.rootPath.dist + 'twig'));
```

Comment:
should be changed to .pipe(gulp.dest(options.styleGuide.source + '/abstractions'));
and kss-example-chroma.twig should be added to .gitignore